GameEngine/Services/Board.cs

```diff
@@ -0,0 +1,85 @@
+﻿using GameEngine.Entities;
+using Repository;
+
+namespace GameEngine
+{
+    /// <summary>
+    /// The class contains all logic of the game engine.
+    /// </summary>
+    public class Board
+    {
+        public string[,] GameBoard;
+
+        /// <summary>
+        /// The class contains all logic of the game engine.
+        /// </summary>
+        public Board()
+        {
+            GameBoard = new string[ConstantsRepository.ColumnsCount, ConstantsRepository.RowsCount];
+            FillBoard();
+        }
+
+        /// <summary>
+        /// Fills the board with animals
+        /// </summary>
+        /// <param name="animals">Animals to fill the board.</param>
+        public void FillBoardWithAnimals(List<Animal> animals)
+        {
+            foreach (var animal in animals)
+            {
+                PutAnimalOnBoard(animal, animal.Letter);
+            }
+        }
+
+        /// <summary>
+        /// Changes current animal with empty cell on the next move.
+        /// </summary>
+        /// <param name="animals">Animals.</param>
+        public void ChangeAnimalWithEmptyCell(List<Animal> animals)
+        {
+            foreach (Animal animal in animals)
+            {
+                GameBoard[animal.CoordinateX, animal.CoordinateY] = ConstantsRepository.EmptyCell;
+            }
+        }
+
+        /// <summary>
+        /// Checks coordinate X and coordinate Y on the ends of board.
```

Comment:
Comment bad. Probably:
Checks whether cell is on board.
Rename method as well.

Reply:
Done.

Reply:
Done.